### PR TITLE
INDY-1184: add warning about old client in TestIndyClusterSetup.md

### PR DESCRIPTION
**Pull request by Toktar**
Changes:

- added warning about old client in TestIndyClusterSetup.md
- changed links from old GSG to new GSG

### CODE REVIEW

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

File: environment/vagrant/training/vb-multi-vm/TestIndyClusterSetup.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 # Setting Up a Test Indy Network in VMs
 
+**WARNING:** Here use deprecated client. Please, use [new client from SDK](https://github.com/hyperledger/indy-sdk/tree/master/cli)
+
 When you're finished working through this document, you will be able to proceed through to the [*Getting
-Started Guide*](../../../../getting-started.md) or if you would like, you may continue setting up an actual Developer Environment connected to a sandbox by following these [instructions](https://github.com/hyperledger/indy-node/blob/master/environment/vagrant/sandbox/DevelopmentEnvironment/Virtualbox/Vagrantfile).
+Started Guide*](https://github.com/hyperledger/indy-sdk/blob/master/doc/getting-started/run-getting-started.md) or if you would like, you may continue setting up an actual Developer Environment connected to a sandbox by following these [instructions](https://github.com/hyperledger/indy-node/blob/master/environment/vagrant/sandbox/DevelopmentEnvironment/Virtualbox/Vagrantfile).
```
Comment (ashcherbakov) [inline environment/vagrant/training/vb-multi-vm/TestIndyClusterSetup.md]: I don't think this is a correct statement. This guide is for the old GSG, not the new one.

File: environment/vagrant/training/vb-multi-vm/TestIndyClusterSetup.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
 # Setting Up a Test Indy Network in VMs
 
+**WARNING:** Here use deprecated client. Please, use [new client from SDK](https://github.com/hyperledger/indy-sdk/tree/master/cli)
```
Comment (ashcherbakov) [inline environment/vagrant/training/vb-multi-vm/TestIndyClusterSetup.md]: I would write it as follows:
**WARNING:** This guideline assumes using of a deprecated CLI and deprecated Getting Started Guide.
Please have a look at [new client from SDK](https://github.com/hyperledger/indy-sdk/tree/master/cli) and [new Getting Started Guide](...) instead.

File: environment/vagrant/training/vb-multi-vm/TestIndyClusterSetup.md (hunk 1 above)
Comment (Toktar) [reply environment/vagrant/training/vb-multi-vm/TestIndyClusterSetup.md]: Will back to the old link. Are here should be link to a new GSG or warning about deprecated this GSG?

Comment (Toktar) [reply environment/vagrant/training/vb-multi-vm/TestIndyClusterSetup.md]: Or warning with a new link in the start willl be enough?

**Comment by Toktar** [comment]
Test this please

**Comment by ashcherbakov** [comment]
test this please

**Pull request merged**